dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -110,26 +110,26 @@ EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
+    CONTROL "Utilisez l'effet de &transition suivant pour les menus et les info-bulles :",
             IDC_EFFAPPEARANCE_ANIMATION, "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 5, 267, 19
     COMBOBOX IDC_EFFAPPEARANCE_ANIMATIONTYPE, 20, 25, 95, 19, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | WS_CHILD | WS_VSCROLL | WS_TABSTOP
-    CONTROL "U&tilisez la méthode suivante pour lisser les bords des polices d'écran :", IDC_EFFAPPEARANCE_SMOOTHING,
+    CONTROL "Utili&sez la méthode suivante pour lisser les bords des polices d'écran :", IDC_EFFAPPEARANCE_SMOOTHING,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 42, 267, 19
     COMBOBOX IDC_EFFAPPEARANCE_SMOOTHINGTYPE, 20, 62, 95, 19, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    CONTROL "Uti&liser de grandes icones", IDC_EFFAPPEARANCE_LARGEICONS, "button",
+    CONTROL "&Utiliser de grandes icones", IDC_EFFAPPEARANCE_LARGEICONS, "button",
             BS_AUTOCHECKBOX | WS_TABSTOP | WS_DISABLED, 10, 80, 267, 19
     CONTROL "&Afficher une ombre sous les menus", IDC_EFFAPPEARANCE_SETDROPSHADOW,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 95, 267, 19
-    CONTROL "Afficher le contenu des fenêtres pendant leur déplacement", IDC_EFFAPPEARANCE_DRAGFULLWINDOWS,
+    CONTROL "A&fficher le contenu des fenêtres pendant leur déplacement", IDC_EFFAPPEARANCE_DRAGFULLWINDOWS,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 110, 267, 19
-    CONTROL "Masquer les lettres soulignées pour la navigation au clavier jusqu'à ce que j'appuie sur la touche Alt",
+    CONTROL "Mas&quer les lettres soulignées pour la navigation au clavier jusqu'à ce que j'appuie sur la touche Alt",
             IDC_EFFAPPEARANCE_KEYBOARDCUES, "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 125, 360, 19
-    CONTROL "Utiliser les menus plats", IDC_EFFAPPEARANCE_FLATMENUS,
+    CONTROL "Utiliser les menus &plats", IDC_EFFAPPEARANCE_FLATMENUS,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 140, 267, 19
-    PUSHBUTTON "Annuler", IDCANCEL, 226, 165, 50, 14
-    DEFPUSHBUTTON "Accepter", IDOK, 172, 165, 50, 14
+    PUSHBUTTON "Annuler", IDCANCEL, 226, 165, 50, 14//FIXME: someone resized the dialog from width 285 to 370 during 0.4.15-dev'ing, that is why those buttons are misplaced now. It would be better to shrink the dlg again, and shorten some texts instead.
+    DEFPUSHBUTTON "OK", IDOK, 172, 165, 50, 14//FIXME: someone resized the dialog from width 285 to 370 during 0.4.15-dev'ing, that is why those buttons are misplaced now. It would be better to shrink the dlg again, and shorten some texts instead. The string that didn't fit the screen was "Mas&quer les lettres soulignées pour la navigation au clavier jusqu'à ce que j'appuie sur la touche Alt" and so I used the shortened "&Cacher les lettres soulignées jusqu'à ce que j'appuie sur la touche Alt" for the backport to allow keeping the dialogs en-US original size
```

Comment:
Is this 285 -> 370 resizing something that happened in all the other languages as well?
This is how it looks like in Win2003 french:
![image](https://github.com/user-attachments/assets/df4f1910-21ce-4b98-a125-894072dfbc0b)

Reply:
No, this was done by some other guy solely for french. And I don't like them to change the dialogs dimensions away from en-US, especially when there is no hard need for that.